server/daos/course_dao.js

```diff
@@ -0,0 +1,52 @@
+// import database
+// import modules
+
+const db = require('../db/db.js');
+const Course = require('../models/course.js');
+
+const createCourse = function (row){
+    return new Course(row.id,row.code,row.name,row.teacher_id);
+}
+
+// it creates the course table
+exports.createCourseTable = function() {
+    return new Promise ((resolve,reject) => {
+        const sql = `CREATE TABLE Courses (id INTEGER NOT NULL PRIMARY KEY, code TEXT NOT NULL UNIQUE,
+                     name TEXT NOT NULL, teacher_id TEXT NOT NULL, FOREIGN KEY(teacher_id) REFERENCES Users(id))`
```

Comment:
teacher_id should be integer?

Reply:
yes cut and paste error, I'll change

Reply:
Maybe we could just add a "year of frequence" field, for fur furthering filtering by the officer (see story 17). But eventually he can also do it manually

Reply:
@s270325 We should probably wait for the csv to see if we will have that information